Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -287,6 +287,29 @@ bool GameListItem::ReadPNGBanner(const std::string& path)
   return true;
 }
 
+const std::string GameListItem::GetFileType() const
+{
+  switch (GetBlobType())
+  {
+  case DiscIO::BlobType::PLAIN:
+    return wxTRANSLATE("Plain");
```

Comment:
Thanks for pointing out that there might be encoding problems - I didn't think of that myself. The current version of the PR has encoding problems because it converts from `std::string` to `wxString` incorrectly, but that will be solved by switching to `_()` because you won't have to do that conversion yourself anymore.